sdk/internal/testframework/README.md

```diff
@@ -0,0 +1,161 @@
+# Azure SDK for Go Recorded Test Framework
```

Comment:
Would be good to mention that by default, the executed tests run in `record` mode if no recordings are available.  And to update recordings, simply delete them and rerun the tests.

Reply:
It's mentioned in the comment on line 38

```go
//mode should be testframework.Playback. This will automatically record if no test recording is available and playback if it is.
recording, err := testframework.NewRecording(context, mode) 
```